dra-evolution/pkg/api/capacity_types.go

```diff
@@ -59,6 +59,15 @@ type ResourcePoolSpec struct {
 	// vendor of the driver.
 	DriverName string `json:"driverName" protobuf:"bytes,3,name=driverName"`
 
+	// SharedCapacity defines the sets of shared capacity consumable by
+	// devices in this ResourceSlice.
+	//
+	// Must not have more than 16 entries.
+	//
+	// +listType=atomic
+	// +optional
+	SharedCapacity []SharedCapacityGroup `json:"sharedCapacity,omitempty"`
+
```

Comment:
Does it make sense to double down on this PR and add the notion of `SharedAttributes` with a similar structure (i.e. a named group of attributes)? Then we can create groups of common attributes that individual devices can reference by name to include (in addition to their own local attributes, which would take precedence on nameing conflicts).

Reply:
I had this in some proposal somewhere, but I an't seem to find it anymore.

Reply:
That's combining this with option 3. I am working through various options, see https://github.com/kubernetes-sigs/wg-device-management/issues/20

Let's see all the options before we decide which to iterate on.